bindings/java/iota-client-java/gradlew

```diff
@@ -1,21 +1,5 @@
 #!/usr/bin/env sh
 
-#
-# Copyright 2015 the original author or authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
```

Comment:
Should this really be removed?

Reply:
hm the one i'm using was automatically generated by gradle... but I will recheck

Reply:
fixed